quickstarts/post-configure-example/README.md

```diff
@@ -0,0 +1,200 @@
+# JBoss CLI PostConfigure example
+
+This example will explain how to create additional JBoss EAP users using the Post Configure feature available on the 
+Container images based on JBoss EAP.
+
+The steps described on this quickstart will work on the following container images:
+
+- RHPAM KIE Server
+- RHDM KIE Server
+- Business Central
+- Business Central Monitoring
+- Decision Central
+- Dashbuilder
+- RHPAM/DM Controller
+
+
+Note that, this procedure can be applied to any kid of post configure actions.
+
+## Preparing the needed files
+
+Before moving forward with the deployments, let's prepare the needed files.
+We need to have the following files:
```

Comment:
@spolti I don't know if that's okay or not but maybe it's interesting to be the second person singular. Like "You need to..."

Reply:
sounds good :)